src/objects.js

```diff
@@ -1,41 +1,52 @@
 const createPerson = (name, age) => {
-  // your code here
+  return {
+    name,
+    age
+  };
 };
 
 const getName = object => {
-  // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  return object.hasOwnProperty(property);
 };
 
 const isOver65 = person => {
-  // your code here
+  return person.age > 65;
 };
 
 const getAges = people => {
-  // your code here
+  return people.map(person => person.age);
 };
 
 const findByName = (name, people) => {
-  // your code here
+  return people.find(person => person.name === name);
 };
 
 const findHondas = cars => {
-  // your code here
+  return cars.filter(car => car.manufacturer === 'Honda');
 };
 
 const averageAge = people => {
-  // your code here
+  const totalAges = people.reduce((sum, person) => sum + person.age, 0);
+  const avAge = totalAges / people.length;
+  return avAge;
 };
 
 const createTalkingPerson = (name, age) => {
-  // your code here
+  return {
+    name,
+    age,
+    introduce: strangersName => {
+      return `Hi ${strangersName}, my name is ${name} and I am ${age}!`;
+    }
+  };
 };
```

Comment:
the `name` and `age` you've referenced in your `introduce` method are the parameters and not the name and age in the returned object. Although this passes the tests fine, if you were to edit the name or age of the person after calling the function, the introduce method would use the original name and age. i.e.
```
const person = createTalkingPerson("Perry", 27);
person.introduce("Chris") // outputs: "Hi Chris, my name is Perry and I am 27"

person.name = "Jasmin"
person.introduce("Chris") // outputs: "Hi Chris, my name is Perry and I am 27"
```
Even though I have changed the person name it is still referring to it by the name the function was originally called with not the name stored in the object. To fix this you'll need to refer to the `name` and `age` in `introduce` using the `this` keyword. 

However, you have used an array function to define your introduce method, so the `this` keyword will refer to the wrong thing. To fix this you'll either need to use the `function` keyword or method shorthand to define this method. The eslint rules prefer you to use method shorthand, so with all that in mind, the final outcome of this looks something like:
```
const createTalkingPerson = (name, age) => {
  return {
    name,
    age,
    introduce(strangersName) {
      return `Hi ${strangersName}, my name is ${this.name} and I am ${this.age}!`;
    }
  };
};
```
Now, if were to do the code above again, this would be the outcome:
```
const person = createTalkingPerson("Perry", 27);
person.introduce("Chris") // outputs: "Hi Chris, my name is Perry and I am 27"

person.name = "Jasmin"
person.introduce("Chris") // outputs: "Hi Chris, my name is Jasmin and I am 27"
```